develop/tutorials/articles/14-themes-and-layout-templates/04-clay-and-themes/00-clay-and-themes-intro.markdown

```diff
@@ -0,0 +1,9 @@
+# Clay CSS and Themes [](id=clay-css-and-themes)
+
+Clay CSS is an extension of Bootstrap's CSS Framework. Bootstrap is by far 
+the most popular CSS framework on the web. Also, it's open source, so anyone can 
+use it. Built with Sass, Clay CSS fills the front-end gaps between Bootstrap 
+and the specific needs of @product@.
+
+These tutorials look briefly at Clay CSS and show you how to use it in your 
+@product@ themes.
```

Comment:
Don't you want to say more about Clay CSS? That it's the web implementation of the Lexicon design language (link)? That for UI consistency, you should implement it?

Reply:
I mention that in the intro to frontend development lexicon and clay section, but I think you'r right. It makes sense to mention that here as well. I'll update and push the changes to this same branch.

Reply:
Updated @sez11a .